packages/dds/tree/src/test/tree/anchorSet.spec.ts

```diff
@@ -588,10 +634,19 @@ function checkEquality(actual: UpPath | undefined, expected: UpPath | undefined)
 	assert.deepEqual(clonePath(actual), clonePath(expected));
 }
 
-function checkRemoved(path: UpPath | undefined, expected: FieldKey = brand("Temp-0")): void {
+function checkRemoved(
+	path: UpPath | undefined,
+	expectedDetachedNodeId: DetachedNodeId | undefined,
+	expected: FieldKey = brand("Temp-0"),
+): void {
 	assert.notEqual(path, undefined);
 	// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 	assert.equal(getDetachedFieldContainingPath(path!), expected);
+	assert.deepEqual(
+		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+		getDetachedUpPath(path!)?.detachedNodeId,
```

Comment:
Looks like this code would be better served by a `tryGetDetachedNodeId` function that returns `DeltaDetachedNodeId | undefined`.